src/main/resources/swagger/api-docs.yaml

```diff
@@ -533,6 +547,32 @@ paths:
       summary: Query to see if you're a library curator
       operationId: isCurator
 
+  /api/library/{workspaceNamespace}/{workspaceName}/discoverableBy:
+    put:
```

Comment:
this is really just setting a library value on an existing workspace. it's broken out into a separate call just for the API because it has different access control checks than the other library fields and these checks are handled when setting the value from the UI.